module/wrtc_helper.go

```diff
@@ -0,0 +1,196 @@
+package module
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"io"
+	"strings"
+	"sync"
+
+	"github.com/pion/sctp"
+	"github.com/pion/webrtc/v3"
+	"github.com/pkg/errors"
+	"go.viam.com/utils"
+
+	"go.viam.com/rdk/logging"
+)
+
+// ConfigureForRenegotiation sets up PeerConnection callbacks for updating local descriptions and
+// sending offers when a negotiation is needed (e.g: adding a video track). As well as listening for
+// offers/answers to update remote descriptions (e.g: when the peer adds a video track).
+//
+// If successful, a Go channel is returned. The Go channel will close when the negotiation
+// DataChannel is open and available for renegotiation.
+func ConfigureForRenegotiation(peerConn *webrtc.PeerConnection, logger logging.Logger) (<-chan struct{}, error) {
```

Comment:
I've put up [this PR](https://github.com/viamrobotics/goutils/pull/273) such that we can avoid adding this file.

Reply:
New tag is published: https://github.com/viamrobotics/goutils/releases/tag/v0.1.71